File: ITRTouchAuth_ObjectiveC/ITRTouchAuth/ITRAuthorisedViewController.m
//
//  ITRAuthorisedViewController.m
//  ITRTouchAuth
//
//  Created by kiruthika selvavinayagam on 11/9/15.
//  Copyright © 2015 kiruthika selvavinayagam. All rights reserved.
//

#import "ITRAuthorisedViewController.h"

@interface ITRAuthorisedViewController ()

@end

@implementation ITRAuthorisedViewController

#pragma mark - lifecycle
- (void)viewDidLoad {
    [super viewDidLoad];
    self.navigationItem.leftBarButtonItems = @[[[UIBarButtonItem alloc] initWithTitle:@"Cancel" style:UIBarButtonItemStylePlain target:self action:@selector(dismissClicked)]];
}
                                                                     
#pragma mark - selector
- (void) dismissClicked {
    [self dismissViewControllerAnimated:YES completion:nil];
}
                                                                     
                                                                    

@end

File: ITRTouchAuth_Swift/ITRTouchAuth_Swift/ITRAuthorisedViewController.swift
//
//  ITRAuthorisedViewController.swift
//  ITRTouchAuth_Swift
//
//  Created by kiruthika selvavinayagam on 11/9/15.
//  Copyright © 2015 kiruthika selvavinayagam. All rights reserved.
//

import UIKit

class ITRAuthorisedViewController: UIViewController {
    
    override func viewDidLoad() {
        super.viewDidLoad()
        self.navigationItem.leftBarButtonItems = [UIBarButtonItem(title:"Cancel", style: UIBarButtonItemStyle.Plain, target: self, action:"dismissClicked")]
    }
    func dismissClicked() {
        self.dismissViewControllerAnimated(true, completion: nil)
    }

}
